client/coral-admin/src/translations.json

```diff
@@ -96,6 +101,7 @@
     },
     "configure": {
       "enable-pre-moderation": "Habilitar pre-moderación",
+      "enable-pre-moderation-text": "tracundeme!",
```

Comment:
english =>"Moderators must approve any comment before it is published"
spanish => "Los moderadores deben aprobar cualquier comentario antes de que sea publicado."